Sources/TuistDependencies/SwiftPackageManager/Utils/SwiftPackageManagerGraphGenerator.swift

```diff
@@ -25,6 +27,8 @@ enum SwiftPackageManagerGraphGeneratorError: FatalError, Equatable {
         switch self {
         case let .unsupportedDependencyKind(name):
             return "The dependency kind \(name) is not supported."
+        case let .missingPathInLocalSwiftPackage(name):
+            return "The local package \(name) does not contain the path in the generated `workspace-state.json` file."
```

Comment:
In which scenario does this happen? If I'm a user and I get that error, can I do something to recover from it or is the only option available to report it as a bug because it's a legit one?

Reply:
When you have a local package the path must always be present, so there is nothing the user can do to recover

Reply:
Changed type to `abort`

Reply:
In that case, the type should be `.bug` because it indicates that something has changed on Apple's side that we should be aware of.